Cargo.toml

```diff
@@ -5,6 +5,18 @@ version = "1.0.0"
 authors = ["Brett Cannon <brett@python.org>"]
 repository = "https://github.com/brettcannon/python-launcher"
 readme = "README.md"
+include = [
+    "Cargo.toml",
+    "/src/",
+    "/benches/",
+    "/tests/",
+    "/completions/",
+    "/docs/control-flow/control_flow.svg",
+    "/docs/man-page/py.1",
+    "/README.md",
+    "/CHANGELOG.md",
+    "/LICENSE",
```

Comment:
The license file is now specified in `Cargo.toml`, so this is no longer necessary as of https://github.com/brettcannon/python-launcher/commit/f90da7e7650be460cfc2ebe7f46e4d9d4c5765bb.

```suggestion
```

Reply:
The `license-file` key is only there for referencing non-standard license texts:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields

Since you're using standard SPDX-recognised licenses, that does not apply here.

Reply:
I think you can read it another way which doesn't make this an exclusive-or situation. The opening definition says:

> The license-file field contains the path to a file containing the text of the license

That suggests it's fine to define even when you have an SPDX expression in `license`. I view the non-standard clause as saying you don't need to panic if you don't have an SPDX expression.

But I would even argue you should always specify `license-file` as 3rd-party notice files need to include the license file verbatim. I've written a tool to generate a 3rd-party notice file and not having the path to the license file is really burdensome.

Reply:
> If a package is using a nonstandard license, then the license-file field may be specified in lieu of the license field.

That's pretty clear to me ... also, almost no other crates do this. The only one I know to use a `license-file` is the [ring](https://crates.io/crates/ring) crate. And there you can see that using `license-file` causes crates.io to list the license as "non-standard".

Reply:
> That's pretty clear to me ... also, almost no other crates do this.

Looks like there's a reason why no one does this: https://github.com/rust-lang/cargo/blob/72ef2bd8ccc354926488164f013e0813f19349a8/src/cargo/util/toml/mod.rs#L1439-L1445 . I have reverted the `license-file` addition in https://github.com/brettcannon/python-launcher/commit/3ddcfb6f6ddc1fd9f08f12de9033696008f03a6d.